packages/addons/addon-depends/dotnet-runtime-depends/aspnet6-runtime/package.mk

```
@@ -11,16 +11,16 @@ PKG_TOOLCHAIN="manual"
 
 case "${ARCH}" in
   "aarch64")
-    PKG_SHA256="b9960c5a343903e38ef84ee8ef00551330b7a64b3fb4d75d8cd2ddd29a23841e"
-    PKG_URL="https://download.visualstudio.microsoft.com/download/pr/e25f7ff2-9932-41dd-b549-5b4409b5a727/d00786aeabad50cd661e959a576f8777/aspnetcore-runtime-6.0.11-linux-arm64.tar.gz"
+    PKG_SHA256="0090d031615b7d49855186cff7e4cbc99636ba664880780f84ce0d5ce25e4b0b"
+    PKG_URL="https://download.visualstudio.microsoft.com/download/pr/8072e219-57e4-48c3-b138-2b4067844ab2/b0712ad06fd0740963bf4ba2eff7f5ea/aspnetcore-runtime-6.0.12-linux-arm64.tar.gz"
```

```suggestion
    PKG_URL="https://download.visualstudio.microsoft.com/download/pr/8072e219-57e4-48c3-b138-2b4067844ab2/b0712ad06fd0740963bf4ba2eff7f5ea/aspnetcore-runtime-${PKG_VERSION}-linux-arm64.tar.gz"
```

wondering if we can reuse PKG_Version ? or is the url changing too much

---

I could do a `sed` on the output from the webpage. But the GUID is different every “update”